model.ipynb

```diff
@@ -0,0 +1,433 @@
+{
```

Comment:
<p><code>tip_amount</code> provides leakage / information about the label <code>tip_percent_greater_15</code> -- so you might want to remove this feature</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/loglabs/ttb/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Done